cylc/flow/task_job_mgr.py

```diff
@@ -1281,6 +1282,33 @@ def _prep_submit_task_job_impl(self, workflow, itask, rtconfig):
             job_d=job_d
         )
 
+    @staticmethod
+    def get_execution_time_limit(
+        config_execution_time_limit: Any
+    ) -> Union[None, float]:
+        """Get execution time limit from config and process it.
+
+        We are making no assumptions about what can be passed in,
+        but passing silently and returning None if the value is
+        not a type convertable to a float:
```

Comment:
Shouldn't it at least log a warning for incoming values that aren't either None or convertible to float, which would clearly be user error. e.g. `12.3x`

Reply:
I was trying to leave well alone and keep it as close to the existing code as possible. But I don't think it's a bad idea to be more specific.